README.md

```diff
@@ -19,34 +21,25 @@ There are two broad categories of biomedical datasets:
-<!---
-Here, we should write maybe 1-3 sentences around our plans for prompting.
--->
-
 ## Contribution Guidelines
 
-There are official guides to contributing to the `datasets` library from Huggingface's  for a [shared dataset](https://huggingface.co/docs/datasets/share_dataset.html) and to [add a dataset](https://huggingface.co/docs/datasets/add_dataset.html). Our guide follows closely from these, with adaptations to suit this intitiative.
+To be guaranteed acknowledgement, participants must implement an *accepted data-loading script* to the bigscience-biomedical collection for **at least 1 dataset**. Explicit instructions are found in [Get started](#Get-started), but the overall criteria to get accepted is as follows: <br>
 
-Contributors must implement an *accepted data-loading script* to the collection for **at least 1 dataset** to be guaranteed acknowledgement. All PRs submitted will be subject to code review prior to acceptance.
+- Implement a data-loading script as a branch
+- PR the branch to the bigscience-biomedical repo
+- If an admin approves the PR, follow the instructions on [uploading to the hub](UPLOADING.md).
 
 Details for contributor acknowledgements and rewards can be found [here](#Thank-you)
 
-## Get started
-
-### Pre-Requisites
+*Optional* If you are looking for the official guides for contributing to the `datasets` library from Huggingface for a [shared dataset](https://huggingface.co/docs/datasets/share_dataset.html) or to [add a dataset](https://huggingface.co/docs/datasets/add_dataset.html), you can find them in the links attached. Our guide follows closely from these, with adaptations to suit this intitiative.
 
-Please do the following before getting started: 
+## Get started!
 
-- [Make](https://huggingface.co/join) an account on 🤗's Hub and [login](https://huggingface.co/login). **Choose a good password, as you'll need to authenticate your credentials**. 
+Please make a github account prior to implementing a dataset; you can follow instructions to install git [here](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git). 
 
-- Join the BigScience Biomedical initiative [here](https://huggingface.co/bigscience-biomedical)
-    - click the "Request to join this org" button in the upper right corner.
+You will also need at Python 3.6+. If you are installing python, we recommend downloading [anaconda](https://docs.anaconda.com/anaconda/install/index.html) to curate a python environment with necessary packages. 
 
-- Make a github account; you can follow instructions to install git [here](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git). 
-
-*Optional* <br>
-- You will also likely need at Python 3.6+. If you are installing python, we recommend downloading [anaconda](https://docs.anaconda.com/anaconda/install/index.html) to curate a python environment with necessary packages. 
+All commands in the guide provided are done through terminal access.
 
 ### 1. Choose a dataset to implement
 
```

Comment:
@hakunanatasha @jason-fries At this moment, I can't comment on the dataset board: https://github.com/orgs/bigscience-workshop/projects/6/views/1  - I'll leave a thread on Slack regarding it

Reply:
python 3.6 is a requirement for `datasets` from what i gather; i'll keep the phrasing but have a comment on this later.

Reply:
also i don't think you can comment on the project board as that's owned by bigscience, you can comment on issues, however.